api/src/main/java/org/pmiops/workbench/api/CloudTaskUserController.java

```diff
@@ -79,17 +78,17 @@ public ResponseEntity<Void> auditProjectAccess(AuditProjectAccessRequest request
                 .filter(
                     project ->
                         project.getParent() == null
-                            || !(ALLOWED_PARENT_IDS.contains(project.getParent().getId())))
+                            || !(ALLOWED_PARENT_IDS.contains(getId(project.getParent()))))
```

Comment:
getParent() returns a string in the new API.  Previously it was a ResourceType (I think?) which had getType() and getId() accessors.